packages/devtools_app/lib/src/memory/memory_heap_tree_view.dart

```diff
@@ -661,11 +664,68 @@ class HeapTreeViewState extends State<HeapTree>
     selectFromSearchField(controller, foundName);
   }
 
+  Widget trackerWidget(MemoryController controller) {
```

Comment:
Is there a use case where we need this instead of manually selecting classes?

Reply:
Re-wrote and eliminated using TreeTable and auto-complete search.